tools/build_jenkins

```diff
@@ -28,6 +28,9 @@ if [[ -z "${ENGBLD_DEST_OUT_PATH}" ]]; then
     export ENGBLD_DEST_OUT_PATH=/public/builds
 fi
 
+export TIMESTAMP=$(TZ=UTC /bin/date "+%Y%m%dT%H%M%SZ")
+export BUILDSTAMP=${TIMESTAMP}
+
```

Comment:
Does this still need to be done early?

Reply:
Unsure if it needs to be done early or not, but it's not strictly necessary. For this change I also reverted all of OS-8408 (60-sec sleep for gcc10) so prior to that the BUILDSTAMP was set this early (and for all of our Jenkins time prior to 8408).